src/en/wizden-staff/release-procedure.md

```diff
@@ -0,0 +1,159 @@
+* If a vote's result is HOTFIX, and the hotfix is not yet completed or approved, we enter a Delay. This should be announced internally so everyone is aware, but no ping is required. Then go do whatever you want, it's sunday! Lizard will be fine.
+
+In case of a tie, recount the Lead Maintainer votes only. If it's still a tie, take the more cautious option and consider the vote a REVERT or HOTFIX, whichever is applicable.
+
+If no blockers remain, go to The Release.
+
+## Delay
+If the hotfixes could not be finished or tested in time, we enter a Delay. The expected maximum duration of the Delay has been agreed upon during the Maintainer meeting, and within this timeframe we should consider the delay acceptable enough not to be "concerned".
+
+If the fix gets completed before the End Time, then any Maintainers present can start the release immediately, but they are also free to just note it and leave someone else to do it later within the accepted Delay.
+
+Available Maintainers should reconvene at the End Time, to decide the continued fate of the blocking PR(s). At this point there will likely be Maintainer consensus established in some way, written decisions from the Lead Maintainers, or Lead Maintainers will be present, but if they aren't, then the Maintainers present decide whether to revert the PR in question or go into overtime waiting for the fix. There is no script for after going into overtime.
+
+> Our level of priority for hitting release targets is currently in general unspecified (but I assume not super high)
+
+## The Release
+
+### 1. Merge Staging into Stable
+
+Do not create a PR.
+Use either [Myra's script](https://gist.github.com/VasilisThePikachu/762b90187a5f9a0cca3249cc335cab31)
+or plain git:
+
+```
+git checkout staging
+git pull upstream staging
+git checkout stable
+git pull upstream stable
+git merge staging
+git push upstream stable
+```
+
+### 2. If there were any hotfixes on Staging that weren't already merged back, merge Stable into Master.
+
+This needs to be done via PR on github (because the master branch is protected from pushing).
+You can merge this PR by yourself immediately, but **DO NOT SQUASH IT**.
+
+If there are merge conflicts at this step, note that you could publish stable before you get bogged down fixing the conflict for master so meanwhile the publish tests can run.
+
+### 3. Run Publish
+
+You can use github-cli to start publish, [Myra's script](https://gist.github.com/VasilisThePikachu/762b90187a5f9a0cca3249cc335cab31) already does this for you.
+
+To do it manually, go to [github's web GUI](https://github.com/space-wizards/space-station-14/actions/workflows/publish.yml) and run the workflow with the stable branch:
+
+![](https://hedgedoc.spacestation14.com/uploads/ff1cdd6c-a2db-42c3-89e1-134bb6809673.jpg)
+
+### 4. If you would like, yell in #general that the release is here, it gets people excited
```

Comment:
Remove

Reply:
Rather than remove, shouldn't we have an actual policy-worded part about announcing an update?

Reply:
Once release happens, the changelog channel gets filled with the changes from the past 2 weeks, but I guess we can do a little announcement on Discourse and link that to codebase changes with a ping of `@contrib-notifs`